src/main.ts

```diff
@@ -59,6 +63,9 @@ export async function run() {
       auth.configAuthentication(registryUrl, alwaysAuth);
     }
 
+    const corepack = core.getInput('corepack') || 'false';
```

Comment:
the input `corepack` is not listed in `/action.yml`
did you forget to add it there?